packages/frontend-shared/src/components/ShikiHighlight.vue

```diff
@@ -72,51 +72,12 @@ shikiWrapperClasses computed property.
   </div>
 </template>
 
-<script lang="ts">
```

Comment:
Moved to it's own file, Vite is more strict and complained about this export.

In general I think it's better if `vue` files only have 1 default export, anyway.